client/components/drop-zone/test/index.jsx

```diff
@@ -37,7 +24,9 @@ describe( 'DropZone', function() {
 	} );
 
 	after( function() {
-		delete window.MutationObserver;
+		if ( global.window && global.window.MutationObserver ) {
```

Comment:
I know that window gets cleaned up and this won't get executed at all; it was semi-conscious decision to clean something up the test created explicitly. 

We're kind of used to not cleaning up the tests since they used to live in different processes; I wanted this to be a good example that cleans up after itself. 

Another argument might be that the consumer of `use-fake-dom` doesn't have to know which variables we are exposing and cleaning up; and should clean up the globals they've introduced.

I'm not feeling wrong strong with either, though.